pkg/config/config.go

```diff
@@ -231,6 +231,7 @@ func init() {
 	BindEnvAndSetDefault("cluster_agent.auth_token", "")
 	BindEnvAndSetDefault("cluster_agent.url", "")
 	BindEnvAndSetDefault("cluster_agent.kubernetes_service_name", "datadog-cluster-agent")
+	BindEnvAndSetDefault("openmetrics_port", "5000")
```

Comment:
As that pprof and goexpvar will also be accessible on this port, WDYT about calling it `metrics_port` instead?